slasher/db/db.go

```diff
@@ -0,0 +1,103 @@
+package db
+
+import (
+	"io"
+	"os"
+	"path"
+	"time"
+
+	"github.com/boltdb/bolt"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var log = logrus.WithField("prefix", "beacondb")
+
+// Database defines the necessary methods for Prysm's eth2 backend which may
```

Comment:
please update this comment to better suit slasher

Reply:
changed